bwl_loot.json

```diff
@@ -422,7 +422,7 @@
   {
     "id": 19374,
     "name": "Bracers of Arcane Accuracy",
-    "role": "Mage = Warlock",
+    "role": "Mage = Warlock = Shadow-Priest",
```

Comment:
i'm not sure if the sheet got updated after i originally made this but this prio was wrong (leering won this last raid)